src/network/networkd-network.c

```diff
@@ -1191,6 +1191,44 @@ int config_parse_required_for_online(
         return 0;
 }
 
+int config_parse_rx_tx_queues(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        Network *network = userdata;
+        uint32_t k;
+        int r;
+
+        if (isempty(rvalue)) {
+                if (streq(lvalue, "TransmitQueues"))
+                        network->txqueues = 0;
+                else
+                        network->rxqueues = 0;
+                return 0;
+        }
+
+        r = safe_atou32(rvalue, &k);
+        if (r < 0 || k > 4096) {
+                log_syntax(unit, LOG_WARNING, filename, line, r, "Failed to parse %s=, ignoring assignment: %s.", lvalue, rvalue);
+                return 0;
+        }
```

Comment:
Please do not pass `r` when `r >= 0`. Also, please reject `k == 0`.
```
if (r < 0) {
    log_syntax(..., r, ...);
    return 0;
}
if (k <= 0 || k > 4096) {
    log_syntax(..., 0, ...);
    return 0;
}
```

Reply:
for `0` it's not going to set any way.